tests/unit/createRouter.ts

```diff
@@ -149,13 +149,14 @@ suite('createRouter', () => {
 	test('dispatch emits navstart event', () => {
 		const router = createRouter();
 
-		let received = '';
+		let received: NavigationStartEvent = null!;
```

Comment:
Best I can tell it silences null check violations. E.g. here you can't normally assign `null` to `received` because it's typed as `NavigationStartEvent`. Writing tests that comply with that behavior is overly cumbersome, hence the silencing.